docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -154,4 +164,32 @@ Map<String, SecurityCheckResult> results = securityService.getCheckExecutor().ad
 
 // adding self defence checks and executing these checks asynchronously
 Map<String, Future<SecurityCheckResult> results = securityService.getAsyncCheckExecutor().addCheck(new customSecurityCheck()).addCheck(SecurityCheckType.<check_type>).addCheck(SecurityCheckType.<check_type>).execute();
-----
\ No newline at end of file
+----
+
+=== Reporting Self-defence checks results via the Metrics service
+
+In order to report the results of Self-defence utilize the link:./metrics[Metrics] service in conjunction with the Security service.
+
+Individual checks can be reported via the `checkAndSendMetric()` method:
+
+[source, java]
+----
+MetricsService metricsService = mobileCore.getInstance(MetricsService.class);
+SecurityService securityService = activity.mobileCore.getInstance(SecurityService.class);
+SecurityCheckResult result = securityService.checkAndSendMetric(SecurityCheckType.IS_ROOTED, metricsService);
+----
+
+Metrics information containing the results of the checks run via an Executor will be automatically sent if the Executor is supplied an instance of the `MetricsService`:
+
+[source, java]
+----
+MetricsService metricsService = mobileCore.getInstance(MetricsService.class);
+SyncSecurityCheckExecutor syncCheckExecutor = SecurityCheckExecutor.Builder.newSyncExecutor(this.getContext())
+    .withSecurityCheck(SecurityCheckType.IS_ROOTED)
+    // other checks...
+    .withMetricsService(metricsService)
+    .build();
+
+// The returned results have already been queued for reporting via the Metrics Service
+Map<String, SecurityCheckResult>  results = executor.execute();
```

Comment:
I would chain the call to `execute` right after the `build` call, to avoid saving the instance of `SyncSecurityCheckExecutor` in its own variable:
`Map<String, SecurityCheckResult>  results = SecurityCheckExecutor.Builder.newSyncExecutor(this.getContext())
...
...
.build().execute()`